config/webpack.common.js

```diff
@@ -0,0 +1,73 @@
+const paths = require("./paths");
+
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+  // Where webpack looks to start building the bundle
+  entry: [paths.src + "/index.js"],
```

Comment:
> Hi @sandrina-p! I'm getting MIME type errors on some of the resources, what could be causing them?
> 
> Edit: I figured out how to integrate images but now there's a conflict with `carousel.js` that I'm not sure how to solve. Another thing is `_flaticon.scss`, I added `"import ./icons/font/_flaticon.scss";` in `index.js ` but the icons still don't display.


I'm not sure, but it seems something related with webpack configs. Some further research about `MIME type errors webpack` might lead you to a solution.